src/msix/PAL/Signature/Win32/SignatureValidator.cpp

```diff
@@ -323,7 +332,7 @@ namespace MSIX
         return(true);
     }
 
-    static PCCERT_CONTEXT GetCertContext(BYTE *signatureBuffer, ULONG cbSignatureBuffer)
+    static PCCERT_CONTEXT GetCertContext(BYTE *signatureBuffer, ULONG cbSignatureBuffer, bool bAllowUnknownSignature)

```

Comment:
I think the intent to skip selfsigned certificates is for the StoreEKU check-- store signed packages should not be self signed. However GetPublisherName should always be OK to handle self-signed certs regardless of options.

Also, don't use Hungarian on the variable name.

So, I would suggest instead explicitly naming it bool allowSelfSignedCert, and have the StoreEKU caller pass in false, and the GetPublisherName pass in true.
If changed that way, we don't need/want to allow the option AllowUnknownSignature because self-signed certs would simply be allowed if they chained to a trusted root.